ex01.exs

```diff
@@ -28,6 +28,23 @@ defmodule Ex01 do
   """
   
   def counter(value \\ 0) do
```

Comment:
Nice.

No deduction, but please keep an eye of layout. In particular, it's nice to have spaces after commas.